### DISCUSSION TITLE
why change pnpm-lock file when i pnpm install?

### DISCUSSION THREAD

**Discussion by Chaocw** [pnpm/pnpm#5136] [Q&A]
<img width="1185" alt="image" src="https://user-images.githubusercontent.com/26131394/182124984-61203314-a415-4070-a3c6-e56051ebaf83.png">

Why it's different from other team members？

**Comment by zkochan** [comment]
Do you use a custom registry?

**Reply by Chaocw** [reply]
this is npm default registry
